libs/common/src/types/dtos/wallet.login.request.dto.ts

```diff
@@ -0,0 +1,47 @@
+import { SignInResponse, SignUpResponse } from '@amplica-labs/siwf';
+import { ApiProperty } from '@nestjs/swagger';
+import { IsNotEmpty } from 'class-validator';
+
+export class WalletLoginRequestDTO {
```

Comment:
Nit: I started to drop the DTO suffix. Thought it was kinda unnecessary. Up to you on if you want to keep my pattern or include it. Happy to change the pattern and put DTO back too.